.devcontainer/Dockerfile

```diff
@@ -309,15 +310,44 @@ RUN git clone https://github.com/OPENAIRINTERFACE/opencord.org.freeDiameter.git
     cmake -DDISABLE_SCTP:BOOL=ON ../ && \
     make -j"$(nproc)" && \
     make install && \
-    cd / && \
-    rm -rf freediameter
+    rm -rf /freediameter /tmp/*
 
 ##### Go language server support for vscode
 RUN go get -v golang.org/x/tools/gopls
 
 #### Update shared library configuration
 RUN ldconfig -v
 
+
+##### Install Python requirements
+RUN pip3 install --quiet --upgrade --no-cache-dir "setuptools==49.6.0"
+
+### install python3-aioeventlet from the magma apt repo (from .circleci/config.yml)
+COPY orc8r/tools/ansible/roles/pkgrepo/files/jfrog.pub /tmp/
+RUN apt-key add /tmp/jfrog.pub \
+    && echo "deb https://facebookconnectivity.jfrog.io/artifactory/list/dev-focal/ focal main" > /etc/apt/sources.list.d/fbc.list \
+    && apt-get update -y  \
+    && apt-get install -y python3-aioeventlet \
+    && rm -f /tmp/* \
+    && rm -rf /var/lib/apt/lists/*  # delete lists that are downloaded by apt-get update
+
+# add patch that is missing in jfrog version of aioeventlet (it only comes with 1 of 2 patches)
+COPY lte/gateway/deploy/roles/magma/files/patches/aioeventlet_fd_exception.patch /tmp/
+RUN patch -N -s -f /usr/local/lib/python3.8/dist-packages/aioeventlet.py < /tmp/aioeventlet_fd_exception.patch \
+    && rm -rf /tmp/*
+
+### install eggs (lte, orc8r)
+COPY /lte/gateway/python/ ${MAGMA_ROOT}/lte/gateway/python/
+WORKDIR ${MAGMA_ROOT}/lte/gateway/python/
+RUN pip3 install --quiet --upgrade --no-build-isolation --no-cache-dir --verbose --editable .[dev] \
+    && rm -rf lte.egg-info
+
+COPY /orc8r/gateway/python/ ${MAGMA_ROOT}/orc8r/gateway/python/
+WORKDIR ${MAGMA_ROOT}/orc8r/gateway/python/
+RUN pip3 install --quiet --upgrade --no-build-isolation --no-cache-dir --verbose --editable .[dev] \
+    && rm -rf orc8r.egg-info
+
```

Comment:
This is probably just me not understanding how our Python setups work, but what do these do? Are these dependencies (?) that cannot be handled with Bazel?

Reply:
Yes, it indeed installs the dependencies of lte and orc8r python code (that is, all packages listed in the respective setup.py files under `install_requires` and `extras_require={ 'dev': [ ...`). Sometime in the future these can and will all be handeled by bazel. For now though, this would be a way to achieve more Python support in the devcontainer instantly.

Background: This is essentially what happens in the make file's `install_egg` target (see https://github.com/magma/magma/blob/master/orc8r/gateway/python/python.mk), just baked directly into the container.